implementations/rust/ockam/ockam_api/tests/token_enrollment.rs

```diff
@@ -50,7 +50,7 @@ async fn admin_can_issue_token(ctx: &mut Context) -> Result<()> {
 }
 
 #[ockam_macros::test]
-async fn admin_cant_accept_token(ctx: &mut Context) -> Result<()> {
+async fn admin_can_accept_token(ctx: &mut Context) -> Result<()> {
```

Comment:
Should it? It'll still get credential as if it's an admin, so it doesn't effect anything?

Reply:
Good point!, right 🤔 ,   this probably was before adding the change so admins get issued credentials too, and didn't revisit.
It's a bit tricky however, we could make that before adding a member or accepting an enrollment token,  we verify the identify trying to do that isn't a valid admin.   But admin' credentials are pretty short-lived so from the user point of view, will be that sometimes it's allowed, and sometimes it isn't.

It's a good think to think of,  how we want to handle this.